roles/znc/handlers/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: enable znc
+  service:
+    name: znc
+    enabled: true
```

Comment:
Why is this a handler? We should only have to do it once, and it is idempotent...